warehouse/internal/repo/schema.go

```diff
@@ -185,8 +185,8 @@ func (sh *WHSchema) Insert(ctx context.Context, whSchema *model.WHSchema) error
 					whSchema.DestinationID,
 					whSchema.DestinationType,
 					tableSchemaPayload,
-					whSchema.CreatedAt.UTC(),
-					whSchema.UpdatedAt.UTC(),
+					now.UTC(),
```

Comment:
If the row already exists, we shouldn't be updating `CreatedAt`, rt ?

Reply:
It's for the insert case, for update we are only setting these: `schema = $5, updated_at = $7,expires_at = $8;`

Reply:
Ok. I didn't understand what is the issue with the current code

Reply:
For all the values that are inserted/updated, I wanted to have the same timestamp instead of calculating it again.